src/app/test/test-component-support.class.spec.ts

```diff
@@ -0,0 +1,94 @@
+import { Component, OnChanges } from '@angular/core';
+import { TestComponentSupport } from './test-component-support.class';
+import { TestBed, inject } from '@angular/core/testing';
+
+let testComponentSupport;
+let testComponentSupportSimple;
+
+@Component({
+  template: `<div
+                app-attr="abc"
+                style="color:blue;"
+                class="class-one class-two"
+                >
+                  <span>test div</span>
+            </div>`
+})
+class FakeComponent implements OnChanges {
+  public ngOnChanges() {
+
+  }
+}
+
+@Component({
+  template: `<div>
+            </div>`
+})
+class FakeSimpleComponent implements OnChanges {
+  public ngOnChanges() {
+
+  }
+}
+
+describe('test component support class', () => {
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        FakeComponent,
+        FakeSimpleComponent
+      ]
+    });
+  });
+  beforeEach(() => {
```

Comment:
Let's put in spacing between `beforeEach` and `it` to make the structure more readable.